### ISSUE TITLE
Report Title in DQ_ConformanceResult of the dataQualityInfo

### ISSUE THREAD

**Issue by LauraAlemany** [inspire-eu-validation/ets-repository#120]
The validator is not detecting the Title of the report, as it is analyzing the previous validator: http://inspire-geoportal.ec.europa.eu/validator2/ 

You can write “Hello” in the title instead of “Reglamento (CE) no 1205/2008 de la Comisión de 3 de diciembre de 2008 por el que se ejecuta la Directiva 2007/2/CE del Parlamento Europeo y del Consejo en lo que se refiere a los metadatos” and there is no error in the Validator. 
However in the previous validator if you don’t write perfectly correct the text in the title, it gerates an error.

¿Is it posible to implement this analysis?

Thank you.

Best Regards,
Laura Alemany

**Comment by cportele** [comment]
I assume this is about the metadata element "resource title". I think that this would require a change in both the Technical Guidelines and the Abstract Test Suite, before this can be implemented in the executable test suite.

Here is the test case that is currently implemented: http://inspire.ec.europa.eu/id/ats/metadata/1.3/iso-19115-19119/title

However, as it is already noted [here](https://github.com/inspire-eu-validation/metadata/tree/1.3/iso-19115-19119#open-questions), this already goes beyond the Technical Guidelines, which do not state any requirements regarding the title.

**Comment by LauraAlemany** [comment]
Thank you Clemens, 

We will wait for the implementation in order to validate that issue.

Best Regards,
Laura

**Comment by klimeto** [comment]
Hi Clemens,
I think Laura meant here INSPIRE metadata element 2.4.1. Conformity whose title shall be given using the gmd:title child element of the citation element with a Non-empty Free Text Element content. For the INSPIRE Implementation Rule documents the value of the title element shall match exactly the official title of the cited document in the language of the metadata.[TG Requirement C.21]. Therefore I think metadata ATS should contain assertions of checking the gmd:title contents agains the values in INSPIRE reference document register.
What do you think?
Best,
Tomas

**Comment by cportele** [comment]
Hi @klimeto - thanks, I see, so the test case would be: https://github.com/inspire-eu-validation/metadata/blob/1.3/iso-19115-19119/ds-conformity.md

However, you refer to the metadata TG version 2.0.1 (for which neither an ATS nor an ETS exists yet). The current ATS and ETS are for version 1.3, which does not have any requirements regarding the title of the conformity element.

I.e., the current ATS/ETS seem to correspond to the technical guidelines. At the same time, developing the ATS/ETS for the new version would address Laura's request.

**Comment by michellutz** [comment]
I agree with @cportele : Version 1.3 of the MD TG do not specify explicitly in a requirement that the title has to be exactly as given in the table in section 2.8.2. 

So if this is checked by the previous MD validator (@aquaglia can you confirm?), that check is probably too strong. 

Since we already have more precise instructions in the new version 2.0.1 of the TG, I don't think it would make sense to introduce an extra requirement in a bug fix version of v1.3.

**Comment by michellutz** [comment]
I would propose to close this issue without making any change to the existing ATS and ETS.

**Comment by aquaglia** [comment]
The INSPIRE Geoportal expects all citations to have a non-empty title, a valid date and a valid date type.
In case of datasets and series, there must be at least the citation of the Interoperability Regulation.
The title is expected to be in the language of the metadata.

MD TGs v.1.3 say:
_TG Requirement 28. In conformance to INSPIRE Directive 2007/2/EC, the metadata shall include information on the degree of conformity with the implementing rules on interoperability of spatial data sets and services._

**Comment by michellutz** [comment]
[2017.4 meeting 2018-02-15] A corrigendum should be published for MD TG v1.3 for the citation to be used (in different languages) for the conformity statement with the implementing rules on interoperability of spatial data sets and services, along the lines of the requirements in v2.0.1 (and the geoportal validator2), and the ATS and ETS should be updated accordingly.

**Comment by michellutz** [comment]
[2017.4 meeting 2018-03-16] The agreed resolution is confirmed. The JRC contractors (Bilbomatica+Guadaltel) will update the ATS, ETS. The JRC will update the TG.

**Comment by inakidiazdecerio** [comment]
Dear @michellutz,
As we agreed in the last meeting we are going to progress on developments / improvements on the ETS and ATS while the technical guidance is modified.

In this specific case, we understand that it is agreed that the title language (Resource language?)  should be the same of metadata language (gmd:MD_Metadata/gmd:language/).

**Comment by michellutz** [comment]
This issue is closely related to #130. The check to be added is whether the title of the CI_Citation element is the title of the Data interoperability IRs in the language of the metadata record. I.e. this is to make the tests for v1.3 consistent with those for v2.0 (and the validator2).
